politeiawww/api/pi/v1/v1.go

```diff
@@ -4,14 +4,97 @@
 
 package v1
 
+import "fmt"
+
 const (
 	// APIRoute is prefixed onto all routes defined in this package.
 	APIRoute = "/pi/v1"
 
 	// RoutePolicy returns the policy for the pi API.
 	RoutePolicy = "/policy"
+
+	// RouteSetBillingStatus sets the record's billing status.
+	RouteSetBillingStatus = "/setbillingstatus"
+)
+
+// Error codes
```

Comment:
This comment doesn't really add any new information.  The `ErrorCodeT` declaration is basically the same thing.  Would prefer you take this out.